packages/ia-components/package.json

```diff
@@ -6,7 +6,7 @@
     "scripts": {
         "storybook": "yarn run test:generate-output && yarn run storybook:start",
         "storybook:start": "start-storybook -p 9001 -c .storybook",
-        "test:generate-output": "mkdir jest-test-utils && jest --json --outputFile=jest-test-utils/jest-test-results.json || true",
+        "test:generate-output": "mkdir -p jest-test-utils && jest --json --outputFile=jest-test-utils/jest-test-results.json || true",
```

Comment:
`-p` makes this fail in Windows
`|| true` makes this fail in Windows

Reply:
@mc2 Do we expect that Windows users to use powershell or cmd or bash emulator?

Reply:
cmd.exe, typically, LCD: lowest common denominator

Reply:
interesting: https://www.npmjs.com/package/run-script-os

But maybe this directory is best pre-created.

Reply:
@mc2 are you on a Windows environment where you could test some code in the command prompt? I'm wondering if we can just rely on `fs` to check for its existence, and then create it if not, like `!fs.existsSync(directory) && fs.mkdirSync(directory);`